obal/data/module_utils/obal.py

```diff
@@ -3,6 +3,22 @@
 """
 import subprocess
 
+def specfile_macro_lookup(specfile, macro_str):
+    """expand a given macro from a specfile"""
+    subproc_output = subprocess.check_output([
```

Comment:
Nit: I think the `command = [\n...\n]\nreturn subprocess.check_output(command)` format might be slightly easier to read.

Reply:
Refactoring.